mcs/class/Facades/netstandard/Makefile

```diff
@@ -28,32 +28,29 @@ endif
 ifeq ($(PROFILE),monodroid)
 # XA implements System.Drawing API inside Mono.Android
 API_BIN_REFS := Mono.Android
-else
+endif
+
+ifeq ($(PROFILE),monotouch)
+API_BIN_REFS := Xamarin.iOS
+endif
+
+ifeq ($(PROFILE),monotouch_tv)
+API_BIN_REFS := Xamarin.TVOS
+endif
+
+ifeq ($(PROFILE),monotouch_watch)
+API_BIN_REFS := Xamarin.WatchOS
+endif
+
+ifeq ($(PROFILE),xammac)
+API_BIN_REFS := Xamarin.Mac
```

Comment:
Commenting here since GitHub won't allow me to comment on the submodule bump:

Unfortunately for xamarin-macios the System.Drawing story is a bit more complicated:

- Xamarin.iOS: Color and KnownColor are in OpenTK-1.0.dll, rest is in Xamarin.iOS.dll -> needs changes
- Xamarin.TVOS: Color and KnownColor are in OpenTK-1.0.dll, rest is in Xamarin.TVOS.dll -> needs changes
- Xamarin.WatchOS: all System.Drawing types are in Xamarin.WatchOS.dll -> fine as is
- Xamarin.Mac: all System.Drawing types are in Xamarin.Mac.dll -> fine as is